docs/concepts/01-what-is-sablier.mdx

```diff
@@ -7,29 +7,105 @@ title: "What Is Sablier?"
+a per-second basis. This innovative approach enables seamless, frictionless transactions and promotes increased
+financial flexibility for users, businesses, and other entities. Sablier makes the passage of time itself the
+trust-binding mechanism, unlocking business opportunities that were previously unavailable.
+
+A good mental model to contrast streaming with conventional payment models is to view the former as "real-time finance"
+or "continuous finance", and the latter as of "discrete finance".
+
+### Permissionless systems
+
+Sablier is rooted in the essential ideas of open access and immutability, deriving inspiration from Ethereum's
+foundational principles and the core values of the DeFi[^1] movement. These concepts are crucial in shaping a future
+where financial services are accessible to everyone, irrespective of their geographical location or economic standing,
+without prejudice or exposure to counterparty risks.
+
+The permissionless design ensures that the protocol's services are open to the public, without any restrictions on who
+can use them. Users have the liberty to establish new streams with any ERC-20 asset, or interact with existing streams
+as they wish. This feature stands in sharp contrast to conventional financial services that frequently impose
+restrictions based on factors such as location, financial status, or age.
+
+As an immutable system, Sablier protocols are not upgradeable, meaning that no party can pause the contracts, reverse
+transactions, or alter the users' streams in any way. This ensures the system remains transparent, secure, and resistant
+to manipulation or abuse.
+
+It is worth noting that the Protocol Admin has the right (but not the obligation) to impose a fee on every stream
+created using specific ERC-20 assets. However, this capability is known to all participants in advance, and to avert
+misuse, the fee percentage is restricted to a range of 0% to 10%.
+
+## Where can I find more information?
+
+For more details on the Sablier Protocols, their features, and potential use cases, explore this documentation site and
+visit the official [Sablier website](https://sablier.com) as well.
 
 :::tip
 
 If you have any questions along the way, please join the #dev channel in our <Link href={links.discord}>Discord
-server</Link>. Our team and members of the community are looking forward to help you.‌ :::
+server</Link>. Our team and members of the community are looking forward to help you.‌
+
+:::
+
+## Release history
+
+- **Lockup V2.2**: The latest version of **The Lockup Protocol**.
+- **Lockup V2.1**: Released in December 2023, it was a minor upgrade over the second version.
+- **Lockup V2.0**: The second big version of **The Lockup Protocol**, released in July 2023.
+- **Lockup V1.1**: Minor upgrade for V1, released in July 2021.
+- **Lockup V1.0**: The first big version of **The Lockup Protocol**, originally released in 2019.
```

Comment:
Yes but I think that should be post v2.0.0 release. Thats when we change the package tethering.

Reply:
You're correct. These should be updated according to package tethering.

Reply:
If we use package versions here, it would confusion with Legacy V1. For example,

```md
- **Lockup v1.2.0**: The latest version of **The Lockup Protocol**.
- **Lockup v1.1.2**: Released in December 2023, it was a minor upgrade over the previous version.
- **Lockup v1.0.2**: The second big version of **The Lockup Protocol**, released in July 2023.
- **Legacy V1.1**: Minor upgrade for V1, released in July 2021.
- **Legacy V1.0**: The first big version of **The Lockup Protocol**, originally released in 2019.
```

Thoughts?

Reply:
i would honestly divide them like this:

- **Flow** (live version - some other notes about flow)
    - latest
    - v1.0.0
- **Lockup** (live version - some notes about lockup)
    - latest
    - v2.0.0
    - v1.2.0
- **Legacy** (outdated version, which was initially released in 2019)
    - v1.0.1
    - v1.0.0

Reply:
yep, let's add subcategories

Reply:
https://github.com/sablier-labs/docs/pull/186/commits/926337b00b562ed5e73afdebcb1fab9dd7c2856b. [Preview](https://v2-docs-git-docs-flow-protocol-sablier.vercel.app/concepts/what-is-sablier#release-history).